include/z64.h

```diff
@@ -379,6 +380,13 @@ typedef struct {
     /* 0xE411 */ char   unk_E411[0x07];
 } MessageContext; // size = 0xE418
 
+typedef enum {
```

Comment:
This should probably go in the environment header file.

Reply:
it gets used by both environment and vrbox, so it has to be global for now

Reply:
Sorry i thought this comment was referring to skybox file

This is for suns song state, which environment doesnt really deal with all too much.
The field for suns state is technically in save context, but it doesnt really feel right there either imo. So I think it can probably stay here for now